examples/client.rs

```diff
@@ -1,14 +1,22 @@
-#![deny(warnings)]
 extern crate hyper;
 
 extern crate env_logger;
 
 use std::env;
-use std::io;
+use std::io::{Read,};  // enables read_to_string
 
 use hyper::Client;
+use hyper::client::response::Response;
 use hyper::header::Connection;
 
+
+// add more parameters here to support adding custom headers
+fn process(client: &Client, url: &str) -> hyper::Result<Response> {
```

Comment:
I am not a fan of splitting this code out into a common function, in my opinion, the code is clearer and more idiomatic if we keep the use of `client` in a simple case like this down below, in main. I think clear variable names and comments will show examples of correct use.